### ISSUE TITLE
Images don't load for example - Interpolation

### ISSUE THREAD

**Issue by sandeepsauravdas** [stackblitz/webcontainer-core#618]
### Describe the bug

Followed the link [here](https://angular.io/generated/live-examples/interpolation/stackblitz.html), for an example for Interpolation from [Angular.io](https://angular.io/guide/interpolation) site. 
![image_2022-03-04_155903](https://user-images.githubusercontent.com/97944427/156746733-25b9dbe4-8c2b-4adf-8a53-19ca90e00545.png)

### Link to the blitz that caused the error

https://stackblitz.com/run?file=src%2Fassets%2Flamp.base64.png

### Steps to reproduce

1. Go to [live Example/Interpolation](https://angular.io/generated/live-examples/interpolation/stackblitz.html)
2. Let the project build and display the site on the right side
3. Check for the images added to site.
![Uploading No Images - Stackblitz - Interpolation.jpg…]()

### Expected behavior

The images on the site should load as is, else an alternative step should be explained how once can view these images on the web container instance of VSCode.

### Parity with Local

- [ ] I have run the project in my local machine and I could not reproduce the issue.

### Screenshots

![image_2022-03-04_165311](https://user-images.githubusercontent.com/97944427/156754817-083702c0-cc1a-43bb-8404-596a44f24c4e.png)
Issue persists on VSCode Local.
![image_2022-03-04_165341](https://user-images.githubusercontent.com/97944427/156754887-26189892-7082-4653-8d68-94a738e70eb6.png)

### Platform

```
Browser name  = Chrome
Full version  = 97.0.4692.99
Major version = 97
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36 OPR/83.0.4254.70
performance.memory = {
  "totalJSHeapSize": 103475632,
  "usedJSHeapSize": 95946108,
  "jsHeapSizeLimit": 2172649472
}
Hash = f79725df77a2d39e41f58b2534bc22660e1ae9b6
```

**Comment by jrvidal** [comment]
@sandeepsauravdas Thanks for the report

> [ ]  I have run the project in my local machine and I could not reproduce the issue.

Can you try to confirm whether the problem appears locally or not?

**Comment by sandeepsauravdas** [comment]
> @sandeepsauravdas Thanks for the report
> 
> > [ ]  I have run the project in my local machine and I could not reproduce the issue.
> 
> Can you try to confirm whether the problem appears locally or not?
> 

Indeed, the issue replicated on the local. However, I figured out the fix. The *.png files, provided for the visuals, are actually corrupt. 

Tried out with other sets of *.png files and the issue was sorted out. 

You might wanna remove those image files there and replace them. That should do it.

PS : I only just checked them a while ago. And, thanks for getting in touch swiftly.
